### PR TITLE
add autos and renamed functions to main

### PR DESCRIPTION
**Pull request by KarlWheezer**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -36,9 +37,10 @@ private CanBus(String value) {
       this.bus_name = value;
     }
   }
+
   public static class FlyWheelConstants {
-    public static final int kRightID = 1;
-    public static final int kLeftID = 2;
+    public static int kRightID = 1;
```
Comment (nowireless) [inline src/main/java/frc/robot/Constants.java:42]: Perhaps undo this.

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -51,25 +53,25 @@ public static class FlyWheelConstants {
...
+    };
+
+    public static double getRPM(double distance) {
       return (227*(Math.pow(distance, 0.578)));
     }
-
- 
   }
+
+  public FlyWheelConstants flyWheelConstants = new FlyWheelConstants();
```
Comment (nowireless) [inline src/main/java/frc/robot/Constants.java:73]: Undo

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -304,8 +300,15 @@ public static class AutoConstants {
...
-}
+
+  public static final Constants kConstants = new Constants();
+    static {
+      if(RobotController.getSerialNumber() == "foobar") {
+        // Practice Bot
+        kConstants.flyWheelConstants.kLeftID = 45;
+      } 
+   }
+}
```
Comment (nowireless) [inline src/main/java/frc/robot/Constants.java:314]: Undo.

**Comment by KarlWheezer** [comment]
resolved changes, ready for push

**Pull request merged**